### PR TITLE
Support external kafka zookeeper

### PR DESCRIPTION
**Pull request by ningyougang**
- [x] support external kafka/zookeeper.

If want to support external kafka/zookeeper, just add below configuration to mycluster.yaml, e.g.
```
zookeeper:
  external: true
  connect_string: xxx,xxx,xxx,xxx:2181,xxx.xxx.xxx.xxx:2182,xxx.xxx.xxx.xxx:2183
  host: xxx.xxx.xxx.xxx

kafka:
  external: true
  connect_string: xxx.xxx.xxx.xxx:9093,xxx.xxx.xxx.xxx:9094,xxx.xxx.xxx.xxx:9095
```
I tested in my local, works well

### CODE REVIEW

File: helm/openwhisk/templates/_helpers.tpl [hunk 1]
```diff
@@ -66,30 +66,36 @@ app: {{ template "openwhisk.fullname" . }}
...
 {{- $kname := printf "%s-kafka" .Release.Name }}
 {{- $kport := .Values.kafka.port }}
 {{- $kubeDomain := .Values.k8s.domain }}
 {{- range $i, $e := until (int .Values.kafka.replicaCount) -}}{{ if ne $i 0 }},{{ end }}{{ $kname }}-{{ . }}.{{ $kname }}.{{ $.Release.Namespace }}.svc.{{ $kubeDomain }}:{{ $kport }}{{ end }}
 {{- end -}}
-
-{{/* host name for server.0 in kafka cluster */}}
-{{- define "openwhisk.kafka_zero_host" -}}
-{{- $kname := printf "%s-kafka" .Release.Name }}
-{{ $kname }}-0.{{ $kname }}.{{ $.Release.Namespace }}.svc.{{ .Values.k8s.domain }}
```
Comment (ningyougang) [inline helm/openwhisk/templates/_helpers.tpl:92]: The template `openwhisk.kafka_zero_host` is not used in other places.

**Comment by dgrove-oss** [comment]
Should we generalize to supporting an external replicated zookeeper and kafka?  Instead of having just a single external host in values.yaml, have a complete external connect string (`h1:p1,h2:p2,....hk:pk`).  When we deploy the internal zookeeper/kafka we support replication.

**Comment by dgrove-oss** [comment]
Also, when you have time please rebase to latest master.  I merged #612 which disables a couple of test cases which have been failing almost all the time in travis.

**Comment by ningyougang** [comment]
@dgrove-oss , already rebased
Let's see whether test cases passed

**Pull request merged**